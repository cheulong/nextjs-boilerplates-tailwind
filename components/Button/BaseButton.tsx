import React from 'react';
import styles from './BaseButton.module.css';

const BaseButton = () => {
  return <button className={styles.wrapper}>BaseButton</button>;
};

export default BaseButton;
