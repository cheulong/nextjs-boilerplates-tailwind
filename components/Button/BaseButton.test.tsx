import { render, screen } from '@testing-library/react';
import BaseButton from './BaseButton';

describe('BaseButton', () => {
  it('renders a BaseButton', () => {
    const { container } = render(<BaseButton />);
    const content = screen.getByText('BaseButton');
    expect(content).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
