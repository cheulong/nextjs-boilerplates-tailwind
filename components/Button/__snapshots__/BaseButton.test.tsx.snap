// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton renders a BaseButton 1`] = `
<div>
  <button
    class="wrapper"
  >
    BaseButton
  </button>
</div>
`;
